cloud/aws/templates/aws_oidc/app.tf

```diff
@@ -34,13 +35,88 @@ module "td" {
       protocol      = "tcp"
     },
   ]
+
+  ecs_task_execution_role_custom_policies = [
+    jsonencode(
+      {
+        "Version" : "2012-10-17",
+        "Statement" : [
+          {
+            "Effect" : "Allow",
+            "Action" : [
+              "secretsmanager:GetSecretValue"
+            ],
+            "Resource" : [
+              aws_secretsmanager_secret.postgres_username_secret.arn,
+              aws_secretsmanager_secret.postgres_password_secret.arn,
+              aws_secretsmanager_secret.app_secret_key_secret.arn,
+              aws_secretsmanager_secret.adfs_secret_secret.arn,
+              aws_secretsmanager_secret.adfs_client_id_secret.arn,
+              aws_secretsmanager_secret.adfs_discovery_uri_secret.arn,
+              aws_secretsmanager_secret.applicant_oidc_client_secret_secret.arn,
+              aws_secretsmanager_secret.applicant_oidc_client_id_secret.arn,
+              aws_secretsmanager_secret.applicant_oidc_discovery_uri_secret.arn,
+            ]
+          },
+          {
+            "Effect" : "Allow",
+            "Action" : [
+              "kms:Encrypt",
+              "kms:Decrypt",
+              "kms:ReEncrypt*",
+              "kms:GenerateDataKey*",
+              "kms:DescribeKey"
+            ],
+            "Resource" : [aws_kms_key.civiform_kms_key.arn]
+          },
+        ]
+      }
+    )
+  ]
+
+  secrets = [
+    {
+      name      = "DB_USERNAME"
+      valueFrom = aws_secretsmanager_secret_version.postgres_username_secret_version.arn
+    },
+    {
+      name      = "DB_PASSWORD"
+      valueFrom = aws_secretsmanager_secret_version.postgres_password_secret_version.arn
+    },
+    {
+      name      = "SECRET_KEY"
+      valueFrom = aws_secretsmanager_secret_version.app_secret_key_secret_version.arn
+    },
+    {
+      name      = "ADFS_DISCOVERY_URI"
+      valueFrom = aws_secretsmanager_secret_version.adfs_discovery_uri_secret_version.arn
+    },
+    {
+      name      = "ADFS_SECRET"
+      valueFrom = aws_secretsmanager_secret_version.adfs_secret_secret_version.arn
+    },
+    {
+      name      = "ADFS_CLIENT_ID"
+      valueFrom = aws_secretsmanager_secret_version.applicant_oidc_client_id_secret_version.arn
+    },
+    {
+      name      = "APPLICANT_OIDC_DISCOVERY_URI"
```

Comment:
Discovery URI and client ID don't need to be a secret - they are also shared with anyone who tries to login, so they are very easy to discover.

Reply:
missed this comment. Agree, those don't need to be secrets.